src/constants/global.ts

```diff
@@ -34,4 +36,27 @@ export const BUTTON_TYPES: {[name: string]: ButtonType} = {
     inverted: 'inverted',
     dark: 'dark',
     disabled: 'disabled'
-};
\ No newline at end of file
+};
+
+export const MENU_ITEMS: MenuItemProp[] = [
```

Comment:
Why isn't this directly in the component? What's the point of having it in this global array?

Reply:
This is also used in the footer.

Reply:
👍